man/oci-image-tool.1.md

```diff
@@ -0,0 +1,43 @@
+% OCI(1) OCI-IMAGE-TOOL User Manuals
+% OCI Community
+% JULY 2016
+# NAME
+oci-image-tool \- OCI (Open Container Initiative) image tools
+
+# SYNOPSIS
+**oci-image-tool** [OPTIONS] COMMAND [arg...]
+
+**oci-image-tool** [--help|-v|--version]
+
+# DESCRIPTION
+oci-image-tool is a collection of tools for working with the [OCI image specification](https://github.com/opencontainers/image-spec).
+
+
+# OPTIONS
+**--help**
+  Print usage statement.
+
+**--debug**
+  Enable debug output
+
+**-v**, **--version**
+  Print version information.
+
+# COMMANDS
+**validate**
+  Validate the given file(s) against the OCI image specification
+  See **oci-image-tool-validate**(1) for full documentation on the **validate** command.
+
+**unpack**
```

Comment:
This language still doesn't make the different between "unpack" and "create" clear.